specification/ai/Face/routes.persondirectory.tsp

```diff
@@ -105,6 +107,7 @@ alias CreateDynamicPersonGroupDescription = """
 alias UpdateDynamicPersonGroupSummary = "Update the name or userData of an existing Dynamic Person Group, and manage its members by adding or removing persons.";
 alias UpdateDynamicPersonGroupDescription = "The properties keep unchanged if they are not in request body.";
 
+@removed(Versions.v1_2)
 interface PersonDirectoryOperations {
```

Comment:
Please double check the API and model of these not-to-be-GAed interface/operation does not appear in Swagger.

Reply:
Looks fine, might because `omit-unreachable-types: true` in `tspconfig.yaml`